build/docker/proxy/assets/etc/nginx/nginx.conf

```diff
@@ -22,6 +22,12 @@ http {
         ssl_certificate      /certificate/certificate.pem;
         ssl_certificate_key  /certificate/certificate.key;
 
+        # Root to static files
+        location /static/  {
+            autoindex    on;
+            root /var/www/capacity-status-api;
```

Comment:
Can we change the directory of the static files to `/application`? This is for consistency as we do that in the other type of containers.

Reply:
So /application/static ? or /application/capacity-status-api/static?